assets/src/js/gravityforms-client-side.js

```diff
@@ -0,0 +1,16 @@
+const p4GFClientside = () => {
```

Comment:
If it is only an invoked function, then you can use an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) here.